core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/CsvTests.kt

```diff
@@ -282,6 +282,32 @@ class CsvTests {
         df.columnNames() shouldBe listOf("Column1", "Column2")
     }
 
+    @Test
+    fun `read empty delimStr or CSV`() {
+        val emptyDelimStr = DataFrame.readDelimStr("")
+        emptyDelimStr shouldBe DataFrame.empty()
+
+        val emptyDelimFile = DataFrame.readDelim(File.createTempFile("empty", "csv").reader())
+        emptyDelimFile shouldBe DataFrame.empty()
+
+        val emptyCsvFile = DataFrame.readCSV(File.createTempFile("empty", "csv"))
+        emptyCsvFile shouldBe DataFrame.empty()
+
+        val emptyCsvFileManualHeader = DataFrame.readCSV(
+            file = File.createTempFile("empty", "csv"),
+            header = listOf("a", "b", "c"),
+        )
+        emptyCsvFileManualHeader shouldBe dataFrameOf("a", "b", "c").fill(0) { "" }
```

Comment:
Why zeros?

Reply:
ah, wait, it's not zeros but 0 rows of empty strings?

Reply:
yes, this generates a DF with 3 named empty String columns

Reply:
i'd prefer checks like above, because it takes time to imagine result of df constructor rather than see expected assertions